nvtabular/loader/backend.py

```diff
@@ -104,37 +104,42 @@ def batch(self, itr):
                 yield current
                 current = []
 
+    def chunk_logic(self, itr):
```

Comment:
overindented here?

Reply:
@benfred I want this in the ChunkQueue class... should it not be in the class itself? I have references to self inside the method.